docs/docs/validators/partial-set-security-for-validators.md

```diff
@@ -120,37 +112,36 @@ interchain-security-pd query provider has-to-validate <provider-validator-addres
 ```
 that can be used by validator with `provider-validator-address` address to retrieve the list of chains that it has to validate.
 
-
-:::tip
-As a validator, the list of chains returned by `has-to-validate` is the list of chains you **should** be validating to avoid
+:::warning
+As a validator, the list of chains returned by `has-to-validate` is the list of chains they **should** be validating to avoid
```

Comment:
```suggestion
For a validator, the list of chains returned by `has-to-validate` is the list of chains the validator **should** be validating to avoid
```